GitHub-Test-4-PyNLPDiscuss/epeck-GitTest4.md

```diff
@@ -0,0 +1,10 @@
+# *Peck-GitTest4*
+
+1. While reading over this what really caught my eye was The Phrase Matcher. This seems to be potentially a very useful tool while organizing and searching for information within documents. 
+
+2. The method that interested me most was picking out certain sentences or phrases from text. Permitting the documents are all correctly spelled and capitalized using this tool will be very helpful.
+
+3. I would be interested on using these methods while looking over historical texts to compare the differences in language and literature at different points through history.
+
```

Comment:
This also interested me and reminded me of the feature that is on windows where "find" shortcut where you type words and it highlights and brings you to the closest word that you typed.